components/builder-api/src/config.rs

```diff
@@ -171,6 +173,12 @@ mod tests {
         client_id = "0c2f738a7d0bd300de10"
         client_secret = "438223113eeb6e7edf2d2f91a232b72de72b9bdf"
 
+        [s3]
```

Comment:
Another thought on how to structure the config - it might be cleaner to call out the backend provider specifically. It lets us provide things like endpoint for minio, vs a region for AWS more naturally. For example:

[S3]
backend = "minio"
access_key = "dev"
secret_key = "secret"
endpoint = "http://localhost:9000"
bucket = "dev"

[S3]
backend = "aws"
access_key = "AWS_KEY"
secret_key = "AWS_SECRET"
region = "AWS_REGION"
bucket = "AWS_BUCKET"

Also as discussed the default should be for minio.

Reply:
yes indeed, perhaps I have this locally but I thought I'd actually already done this! I will validated and get this in. I ran into this myself last week!

Reply:
Yeah it appears I hadn't pushed this up! Also I've duplicated these settings into the default.toml

Reply:
👍